src/qaqc/download_and_flag_fxns/add_spec_flag.R

```diff
@@ -0,0 +1,19 @@
+# Adding flags related to sensor specification ranges; these are instances where
+# a value exceeds the manufacturer specified limits.
+
+add_spec_flag <- function(df){
```

Comment:
question: can the stuff from the yaml be added to the look up table?

Reply:
I was thinking that too! But, it would be really repetitive since it will be the same value across all sites. Either way, I think we should store both of this data in the same place...